### DISCUSSION TITLE
OGC MF-JSON Prism Support

### DISCUSSION THREAD

**Discussion by mdgilene** [anitagraser/movingpandas#432] [Q&A]
Hi, I'm currently working on a project that is using standard GeoJSON for feature storage and manipulation. However, we have been asked to investigate moving to MF-JSON as our data often has a temporal aspect as well. Upon doing some research it looks like MF-JSON has been broken into 2 sub-categories:

1. MF-JSON Trajectory
    This category seems to be largely the same as standard GeoJSON and is designed for moving points over time. i.e. no geometry deformation or rotation
2. MF-JSON Prism
    This category enables support for complex geometries over time which may change shape, size, or rotation.

We are already using Geopandas for GeoJSON, and it seems like this library is exactly what we need for the first case and handles Trajectories very well. My question is does this library have any support for the second category of MF-JSON? I have looked through the docs and the code and don't seem to see anything about Prism support.

If the answer to the above is no, are there any plans to implement support for MF-JSON Prism in the future?

Thanks!

**Comment by anitagraser** [comment]
Hi Matt. Prism support isn't on the roadmap right now, but if you're interested in giving it a try, I would appreciate if you can report your findings.
